app/templates/single_resource/index.html

```diff
@@ -17,29 +24,48 @@ <h2 class="ui dividing header">Resources</h2>
                 </div>
             </div>
         </div>
-        <div style="overflow-x: scroll;">
-            <table class="ui searchable sortable unstackable selectable celled table">
-                <thead>
-                    <tr>
-                        <th>Name</th>
-                        <th>Address</th>
-                        <th>Latitude</th>
-                        <th>Longitude</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    {% for resource in resources %}
+            <!--waiting for functionality-->
```

Comment:
and let's comment this stuff back in except for the dropdown menu `<div class="ui basic floating dropdown button">` block